lib/entry.js

```diff
@@ -209,4 +136,74 @@ Ep.runModuleSetters = function (module) {
   }
 };
 
+function call(setter, name, value, callback) {
+  if (name === "__esModule") {
+    // Ignore setters asking for module.exports.__esModule.
+    return;
+  }
+
+  if (value !== value) {
+    value = NAN;
+  } else if (typeof value === "undefined") {
+    value = UNDEFINED;
+  }
+
+  setter.last = setter.last || Object.create(null);
+
+  if (setter.last[name] !== value) {
+    // Only invoke the callback if we have not called this setter
+    // (with a value of this name) before, or the current value is
+    // different from the last value we passed to this setter.
+    return callback(setter, name, value);
```

Comment:
I think the third argument should be the original `value` passed into this function, even if it was replaced by `NAN` or `UNDEFINED` for the `setter.last[name]` logic.